src/zcash/circuit/note.tcc

```diff
@@ -96,11 +96,6 @@ public:
         a_sk->generate_r1cs_constraints();
         rho->generate_r1cs_constraints();
 
-        // TODO: These constraints may not be necessary if SHA256
-        // already boolean constrains its outputs.
-        a_pk->generate_r1cs_constraints();
-        commitment->generate_r1cs_constraints();
-
```

Comment:
Preaching to the converted there, but comments are (a little) better than nothing.